README.md

```diff
@@ -218,12 +218,38 @@ Measurable are gathered and observed using prometheus/grafana.
 
 ### Data relationships
 
-Contracts can be linked together by creating a contract of type "link" that references both contracts and describes their relationship. Relationships can be traversed when querying data using the `$$links` syntax.
+Contracts can be linked together by creating a contract of type `link` that references both contracts and describes their relationship. Relationships can be traversed when querying data using the `$$links` syntax.
+
+`Link` contracts are described by a [`relationship` contract](./lib/contracts/relationship.ts). Relationship contracts relate two contracts which are specified using the `from` and `to` properties, and describe two directions, from `from` to `to` using the `name` property, and from `to` to `from` using the `inverseName` property.
+
+Example:
+
+```ts
+{
+  slug: `relationship-message-is-attached-to-issue`,
+  type: 'relationship@1.0.0',
+  name: 'is attached to',
+  data: {
+  	inverseName: 'has attached element',
+  	title: 'Message',
+  	inverseTitle: 'Issue',
+  	from: {
+  		type: 'message@1.0.0',
```

Comment:
Can drop the version, just leave the type slug

Reply:
Changed.